### DISCUSSION TITLE
새로고침 주기에 대해 질문입니다.

### DISCUSSION THREAD

**Discussion by aodhan77** [changdoc/naver-vaccine-macro#172] [Q&A]
만들어주신 코드 정말 감사히 쓰겠습니다.

문의드릴께 하나 있습니다.

병원 개별 신청페이지 2군데정도 탭을 띄워서, 해당 페이지에서 돌리게 되면 새로고침 시간을 늘려야 하는건 알겠습니다.

네이버 플레이스(지도)에서 예약시작을 했을때 -> graphql의 병원이 4개정도 지정되어있을경우 이 경우도 새로고침 시간을 늘려야 하나요?

**Comment by changdoc** [comment]
네이버 지도에서 동작하는 건 신청 페이지에 짧은시간 내에 여러번 접근하는 형식이 아니라서 신청 페이지가 차단되는 상황은 발생하지 않는 것으로 확인 했습니다.
